release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1381,6 +1395,15 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Storage*
 
+* Previously, the recycler-pod template was incorrectly placed in the kubelet static manifest directory. This incorrect location produced kubelet static pod log messages indicating a recycler static pod start failure. With this update, the misplaced recycler-pod template has been removed from the kubelet static pod manifest directory. As a result, the error messages no longer appear. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896226[*BZ#1896226*])
+
+* Previously, the Local Storage Operator (LSO) could claim disks belonging to other provisioners because busy disks were erroneously detected as free. Disks are now checked for bind-mounts so that the LSO cannot claim those disks. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929175[*BZ#1929175*])
+
+* Previously, the Local Storage Operator (LSO) would attempt to create a persistent volume (PV) with an invalid label value, because the device-id contained unsupported characters such as `:`. This has been corrected by moving the device information from labels to annotations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1933630[*BZ#1933630*])
+
+* Previously, the Local Storage Operator was not cleaning up persistent volumes (PVs) since the deleter was not being enqueued correctly. This caused PVs to remain in the `released` state. PVs are now enqueued correctly so that they delete properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937145[*BZ#1937145*])
+
+* Previously, FibreChannel was incorrectly unmounted from a node when a pod was deleted. This happened when a different pod that used the volume was deleted in the API server when kubelet on the node was not running. With this update, FibreChannel volumes correctly unmount when a new kubelet starts. Additionally, the volume cannot be mounted to multiple nodes until the new kubelet fully starts and cofnrims that the volume is unmounted, which ensures that FibreChannel volumes are uncorrupted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954509[*BZ#1954509*])
```

Comment:
s/FibreChannel/Fibre Channel in 3 places (I don't see FibreChannel in the docs)
Should this be Fibre Channel volume(s), as with the second reference? (_Fibre Channel volumes correctly unmount..._) 
_Previously, any Fibre Channel volume was..._

/when kubelet/when the kubelet